app/models.py

```diff
@@ -2907,7 +2914,7 @@ class AdminAuditLog(Base):
     action = sa.Column(sa.Integer, nullable=False)
     model = sa.Column(sa.Text, nullable=False)
     model_id = sa.Column(sa.Integer, nullable=True)
-    data = sa.Column(sa.JSON, nullable=True)
+    data = sa.Column(sa.JSON, nullable=False)
```

Comment:
I don't see a migration script for making the data non-nullable, should we generate one?

Reply:
In this case we also handle the existing rows that have `data=null`

Reply:
the model in the db is non-nullable. This is bugfix. It's the same issue as the first comment in models.py where I add a `data={}` to the `logged_as_user` function